pkg/apiserver/authz.go

```diff
@@ -92,10 +97,12 @@ func NewAuthorizerFromAuthorizationConfig(authorizationModes []string, authoriza
 				return nil, errors.New("ABAC's authorization policy file not passed")
 			}
 			abacAuthorizer, err := abac.NewFromFile(authorizationPolicyFile)
+			authorizerLock := &abac.PolicyAuthorizer{PL: &abacAuthorizer, Lock: &sync.RWMutex{}}
```

Comment:
Whether this reloads or not needs to be configurable, not always true.

For instance, when running apiserver in HA configurations, administrators may not want this behavior.